src/sc_keeper/parameters.py

```diff
@@ -173,6 +173,18 @@
     ),
 ]
 
+direction = Annotated[
+    Optional[List[str]],
+    Query(
+        title="Direction",
+        description="Direction of the Internet traffic.",
+        json_schema_extra={
+            "category_id": FilterCategories.TRAFFIC,
+            "enum": ["inbound", "outbound"],
```

Comment:
let's import `TrafficDirection from fields, then:

```suggestion
            "enum": [e.value for e in TrafficDirection],
```